src/qibolab/pulses/pulse.py

```diff
@@ -113,7 +103,7 @@ class Acquisition(_PulseLike):
     """Duration in ns."""
 
 
-class _Readout(_PulseLike):
+class Readout(_PulseLike):
```

Comment:
Indeed, the plan is to always use `Readout` to translate `MZ`, also because it retains more information (the two operations being related).
The only bit I'm uncertain about is whether I should add a `Delay` on the probe channel, to tell that it's busy and a separate probe `Pulse` would not be supported. For the time being, I decided to pretend the actual probe channel (in the sequence) to be dead if any `Readout` operation is present.

For circuits, you should always use 2., as the two operations are related. 1. is there to be available for other calibration purposes (e.g. calibrating the electronics itself, without any connected chip).

2. should be always supported by the drivers, possibly translating it to 1., if needed anyhow or easier (it will be the `.as_readouts` inverse, but that's much more intuitive, as it boils down to unpack the `Readout` content in the sequence, just taking care of adding suitable delays in the probe).
1. may be supported, if possible, or ignored otherwise.